app/kubernetes-ingress-controller/1.0.x/guides/prometheus-grafana.md

```diff
@@ -94,7 +96,8 @@ dashboards:
 To install Grafana, execute the following:
 
 ```bash
-$ helm install stable/grafana --name grafana --namespace monitoring --values http://bit.ly/2FuFVfV --version 5.0.8
+$ helm repo add grafana https://grafana.github.io/helm-charts
+$ helm install grafana grafana/grafana --namespace monitoring --values http://bit.ly/2FuFVfV --version 5.0.8
```

Comment:
Same as for prometheus. Why not use the newest version?

(applies to all copies of this)

Reply:
Ditto https://github.com/Kong/docs.konghq.com/pull/3288#discussion_r721699614